CMakeLists.txt

```diff
@@ -15,15 +15,31 @@ enable_testing()
 
 include (cmake_modules/FinalFile.cmake)
 
-if (NOT CMAKE_BUILD_TYPE)
```

Comment:
Wait a second, I think I did the merge tool backwards on this file.  It was a conflict and I think I selected the wrong side.

https://github.com/supercollider/supercollider/blob/master/CMakeLists.txt#L18